on-prem-docs/separating-backend-services.md

```diff
@@ -0,0 +1,55 @@
+# Separating Backend Services (minio/postgresql)
+
+The on-prem-builder install.sh script now supports separation of the backend components i.e datastore and minio server onto different nodes.
+You can now have a setup where the postgresql service runs on one node and minio runs on another. Both can be designed to communicate to one another and to a third or more nodes running the front-end services. 
```

Comment:
Both can be configured to communicate with other nodes running the front-end services.